openedx/core/djangoapps/user_api/accounts/toggles.py

```diff
@@ -31,14 +31,12 @@ def should_redirect_to_order_history_microfrontend():
 # .. toggle_description: Supports staged rollout of a new micro-frontend-based implementation of the account page.
 # .. toggle_use_cases: temporary, open_edx
 # .. toggle_creation_date: 2019-04-30
-# .. toggle_target_removal_date: 2020-12-31
+# .. toggle_target_removal_date: 2021-12-31
 # .. toggle_warnings: Also set settings.ACCOUNT_MICROFRONTEND_URL and site's ENABLE_ACCOUNT_MICROFRONTEND.
```

Comment:
1. 
```
.. toggle_warnings: Also set settings.ACCOUNT_MICROFRONTEND_URL.
```
2. Add something like: "Can be overridden using site's ENABLE_ACCOUNT_MICROFRONTEND." to the `toggle_description`.

Reply:
@robrap already fixed